Tests/AppTests/MetricsTests.swift

```diff
@@ -61,18 +61,6 @@ extension AllTests.MetricsTests {
     @Test func versions_added() async throws {
         try await withApp { app in
             // setup
-            let initialAddedBranch = try #require(
-                AppMetrics.analyzeVersionsAddedCount?.get(.versionLabels(kind: .branch))
-            )
-            let initialAddedTag = try #require(
-                AppMetrics.analyzeVersionsAddedCount?.get(.versionLabels(kind: .tag))
-            )
-            let initialDeletedBranch = try #require(
-                AppMetrics.analyzeVersionsDeletedCount?.get(.versionLabels(kind: .branch))
-            )
-            let initialDeletedTag = try #require(
-                AppMetrics.analyzeVersionsDeletedCount?.get(.versionLabels(kind: .tag))
-            )
```

Comment:
We don't need to baseline the counts anymore because we have a new, 0-based prometheus client on each test run.